app/views/shared/export/_plan_coversheet.erb

```diff
@@ -31,6 +31,10 @@
     <p><b><%= _("ID: ") %></b><%= @plan.id %></p> <br>
```

Comment:
We should remove this section where we shou Id with the @plan.id as we are using the correct identifier as you added below.

Reply:
Sure I can do this, but will the user like to maintain this to retrieve the actual plan?

Reply:
How do you mean? Like using the plan id presented here to form a URL?

Reply:
Yes will the user want to use this id to check the related plan?

Reply:
I believe that the plan here was to show the identifier all along. Even on the user interface, you changed it to be the identifier instead of the id. The Id was just used to initialize that value. 

We shouldn't expect the users to know the plan id to reference it back to the application as it is not in the documentation and bad practice in general. If this was the case we could add an URL. For now we should just remove it.

Reply:
make sense!